lib/common/strings.c

```diff
@@ -600,3 +600,26 @@ crm_strdup_printf(char const *format, ...)
     va_end(ap);
     return string;
 }
+
+bool
+pcmk__split_range(const char *srcstring, char separator, char **name, char **value)
```

Comment:
If we're going to do this, we may as well go all out :)

Let's call it pcmk__parse_ll_range(), assume separator is '-' (can you think of any other single-character range separator?), and use long long* for the start and end (using scan_ll(), which will avoid needing strchr()).

Reply:
Comma and colon are about the only other ones that look reasonable.

Reply:
True, but it's private API, so let's assume '-' for now and we can add it back as an argument if the need arises